src/validate.js

```diff
@@ -67,7 +67,11 @@ async function validator(options) {
     try {
       ajv.addSchema(schema);
       debug('Loaded schema with id of "%s"', schema.id);
-      schemas.push({name: name, schema: schema});
+      let content = JSON.stringify(schema, undefined, 4);
+      if (!content) {
+        throw new Error('Schema %s has invalid content!', name);
+      }
+      schemas.push({name: name, content});
```

Comment:
Schemas already contains an id property, so I think introducing another name is just going to confuse things...

Can't schemas just be a list of schemas?